packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -615,6 +615,59 @@ function getRank(
   return getRankOrder(memberGroups, orderConfig);
 }
 
+/**
+ * If there are more than 1 consecutive same ranked member in 1 class, those members are put in an array & an array of these members' array is returned as a result. If, for example, the memberSet parameter looks like the below example - (contd. after the example tag)
+ * @example
+ * ```
+ * interface Foo {
+      [a: string]: number;
+ 
+      a: x;
+      B: x;
+      c: x;
+ 
+      c(): void;
+      B(): void;
+      a(): void;
+ 
+      (): Baz;
+ 
+      new (): Bar;
+    }
+ * ```
+  The resulting array will look like this - [[a, B, c], [c, B, a]].
+ * @param memberSet The members in the class of the source code on which the grouping operation will be performed.
+ * @param memberType The member type order listed in the eslint config file.
+ * @param supportsModifiers it'll get passed to getRank()
+ * @returns the array of groups of members
+ */
+function groupMembersByType(
+  memberSet: Member[],
```

Comment:
[Naming] "Set" is normally used to refer to the actual `Set` object, or sometimes the general concept of a unique set of things. So it's a little confusing to see it written here.

The parameter passed to `checkAlphaSortForAllMembers` is called `members`. How about using that as the name? Or `allMembers` if you find `members` to be too broad? Something like those?

```suggestion
  members: Member[],
```

I do see that this is an existing name from the `checkOrder` function. Which came from a PR I approved. Now I know better. 😄